src/routes/common/decorators/utils.spec.ts

```diff
@@ -1,13 +1,14 @@
 import { faker } from '@faker-js/faker';
 import { getRouteUrl } from '@/routes/common/decorators/utils';
+import { Request } from 'express';
 
 describe('utils tests', () => {
   describe('getRouteUrl tests', () => {
     const request = {
       get: jest.fn(),
       originalUrl: faker.system.filePath(),
       protocol: faker.internet.protocol(),
-    } as unknown as any;
+    } as unknown as jest.MockedObjectDeep<Request>;
```

Comment:
```suggestion
    } as jest.MockedObjectDeep<Request>;
```

In fact, I think we could revisit some `as unknown` casts we are doing through the code (I can tackle this in a separate PR) 🙂

Reply:
Iremoved this particular instance in 771fef093993851cd5fcdf1b3161d3ac4af4fb95 but we definitely should tackle the others.